src/_guides/language/effective-dart/style.md

```diff
@@ -420,27 +415,26 @@ main() {
 
 anotherDeclaration() { ... }
 {% endprettify %}
-</div>
 
 
 ### DON'T place a space between the declared name of a method, operator, or setter and its parameter list.
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/style_good.dart (args-etc-no-spaces)"?>
 {% prettify dart %}
-bool convertToBool(arg) { ... }
-bool operator ==(other) { ... }
+log(arg) { ... }
```

Comment:
Why did this change?

Reply:
In this case, the method return type annotation (`bool`) seemed to distract from the "DON'T place a space between the declared name of a method, operator, or setter and its parameter list" guideline, so I switched to a method whose return type doesn't need to be shown.